modules/02_welfare/ineqLognormal/datainput.gms

```diff
@@ -44,6 +44,7 @@ if ((cm_emiscen ne 1),
     Execute_Loadpoint 'input_bau' p02_taxrev_redistr0_ref=v02_taxrev_Add.l;
     Execute_Loadpoint 'input_bau' p02_cons_ref=vm_cons.l;
     Execute_Loadpoint 'input_bau' p02_energyExp_ref=v02_energyExp.l;
+    Execute_Loadpoint 'input_bau' p02_damageFactor_ref=vm_damageFactor.l;
```

Comment:
are you sure this is correct with `input_bau` here? Because if that is correct, I'm not sure whether [this code](https://github.com/remindmodel/remind/blob/2758f32fe4907ad33f575f45aff497580f949012/scripts/start/readCheckScenarioConfig.R#L84-L94) is correct, that may potentially remove the `path_gdx_bau` as not needed if the 45 / 46 realizations don't need it.

Reply:
I changed it to input_ref. Didn't know about this code.